### DISCUSSION TITLE
usefulib 1.0.5

### DISCUSSION THREAD

**Discussion by hamdivazim-archive** [hamdivazim-archive/usefulib#16] [Announcements]
### Fixes
* Improvements to contribution guidelines
* Optimise test scripts

### New Additions
* `denery_to_ternary()` by @AtomicCodeLegend (#12)
* `calculate_fibbonaci()` by @MKM12345 (#14)
* `is_palindrome()` by @MKM12345 (#14)
* `convert_base()` by @ShadowStrike-Atomiser (#15)

### What's Changed
* denary_to_ternary() by @AtomicCodeLegend by @AtomicCodeLegend in https://github.com/hamdivazim/usefulib/pull/12
* convert_base() by @ShadowStrike-Atomiser in https://github.com/hamdivazim/usefulib/pull/15

### New Contributors
* @AtomicCodeLegend made their first contribution in https://github.com/hamdivazim/usefulib/pull/12
* @ShadowStrike-Atomiser made their first contribution in https://github.com/hamdivazim/usefulib/pull/15

**Full Changelog**: https://github.com/hamdivazim/usefulib/compare/v1.0.4...v1.0.5

<hr /><em>This discussion was created from the release <a href='https://github.com/hamdivazim/usefulib/releases/tag/v1.0.5'>usefulib 1.0.5</a>.</em>

**Comment by AtomicCodeLegend** [comment]
yay
